CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to Stryker Dashboard
+
+This is the contribution guide for Stryker Dashboard. Great to have you here! Here are a few ways you can help make this project better.
+
+## Creating issues
+
+Do you have an idea for a feature or have you found a bug? Please create an issue so we can talk about it!
+
+## Adding new features
+
+New features are welcome! Either as requests or proposals.
+
+1. Please create an issue first, so we know what to expect from you.
+2. Create a fork on your GitHub account.
+4. Make the changes or additions in your fork.
+5. Please create or edit unit tests or integration tests.
+6. Run the tests and verify they pass.
+7. Update documentation.
+8. [Create](https://github.com/stryker-mutator/stryker-dashboard/compare) a Pull Request.
+
+## Running the Stryker Dashboard locally
+
+Please check out the [README.md](https://github.com/stryker-mutator/stryker-dashboard/blob/master/README.md) to help you getting started with running the dasbhoard locally.
+
+## Working on existing issues
+
+Do you want to contribute to the Stryker Dashboard, but don't know where to start? 
+
+Rest assured, we have prepared a few issues for this purpose. Most of the issues labeled with [`hacktoberfest`](https://github.com/stryker-mutator/stryker-dashboard/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest) are suited to be picked up by pretty much anyone!
+
+If you are still not sure where to start or if you need help implementing anything, please reach out to us on [Slack](https://join.slack.com/t/stryker-mutator/shared_invite/enQtOTUyMTYyNTg1NDQ0LTU4ODNmZDlmN2I3MmEyMTVhYjZlYmJkOThlNTY3NTM1M2QxYmM5YTM3ODQxYmJjY2YyYzllM2RkMmM1NjNjZjM) or simply comment on the issue that you would like to work on.
```

Comment:
The Slack doesn't have a channel for Stryker dashboard. I think it's easier for prospect contributors to have a dedicated channel for the project. @nicojs